packages/core/src/services/evaluations/update.ts

```diff
@@ -0,0 +1,41 @@
+import { eq } from 'drizzle-orm'
+
+import { EvaluationDto } from '../../browser'
+import { database } from '../../client'
+import { Result, Transaction } from '../../lib'
+import { compactObject } from '../../lib/compactObject'
+import { evaluations, llmAsJudgeEvaluationMetadatas } from '../../schema'
+
+export async function updateEvaluation(
+  {
+    evaluation,
+    name,
+    description,
+    metadata = {},
+  }: {
+    evaluation: EvaluationDto
+    name?: string
+    description?: string
+    metadata: Record<string, unknown>
+  },
+  trx = database,
+) {
```

Comment:
@csansoon never pass ids to services that write to db, always full models. The reason is to force the consumer to use repositories and thus enforce tenancy.